event_routing_backends/handlers.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module contains the handlers for signals emitted by the analytics app.
+"""
+import json
+import logging
+
+from django.dispatch import receiver
+from eventtracking.backends.event_bus import EventBusRoutingBackend
+from eventtracking.processors.exceptions import EventEmissionExit
+from eventtracking.tracker import get_tracker
+from openedx_events.analytics.signals import TRACKING_LOG_EVENT_EMITTED
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(TRACKING_LOG_EVENT_EMITTED)
+def send_tracking_log_to_backends(
+    sender, signal, **kwargs
+):  # pylint: disable=unused-argument
+    """
+    Listen for the TRACKING_LOG_EVENT_EMITTED signal and send the event to the enabled backends.
+
+    The process is the following:
+
+    1. Unserialize the tracking log from the signal.
+    2. Get the tracker instance to get the enabled backends (mongo, event_bus, logger, etc).
+    3. Get the event bus backends as those are the interested in the signals and multiple can be configured.
+    4. Process the event with the event bus backend.
```

Comment:
I'm not sure what this means. There are potentially several `EventBusRoutingBackend` instances, right? It might be helpful to say that "process" in this context means "transform to xapi or caliper", at least I think it does.

Reply:
Yes, one for xAPI and for Caliper